ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master.yaml

```diff
@@ -1,28 +1,8 @@
 base_images:
-  assisted-installer:
-    name: ocm-2.6
-    namespace: edge-infrastructure
-    tag: assisted-installer
-  assisted-installer-controller:
-    name: ocm-2.6
-    namespace: edge-infrastructure
-    tag: assisted-installer-controller
-  assisted-service:
-    name: ocm-2.6
-    namespace: edge-infrastructure
-    tag: assisted-service
-  assisted-test-infra:
-    name: ocm-2.6
-    namespace: edge-infrastructure
-    tag: assisted-test-infra
   coreos-centos-stream8:
     name: stream8
     namespace: coreos
     tag: "8"
-  dev-scripts:
-    name: test
-    namespace: ocp-kni
-    tag: dev-scripts
```

Comment:
I think we're going to need dev-scripts image, while very likely centos-stream8 not

Reply:
This is true. I put it back in but it is likely the CI bots will clean it up for us because there is a job that removes unused images from the inputs stanza. 😄